harness/ts/main.ts

```diff
@@ -38,14 +47,16 @@ async function run() {
   for (const featureAndTaskQueue of opts.featureAndTaskQueues) {
     // Split
     const colon = featureAndTaskQueue.indexOf(':');
-    const featureDir = featureAndTaskQueue.substring(0, colon);
+    const cpos = colon == -1 ? undefined : colon;
+    const featureDir = featureAndTaskQueue.substring(0, cpos);
     const taskQueue = featureAndTaskQueue.substring(colon + 1);
 
     let runner;
     try {
       // Find the source
       const source = sources.find((s) => s.relDir === featureDir);
       if (!source) {
+        // noinspection ExceptionCaughtLocallyJS
```

Comment:
What's inspecting this?

Reply:
IntelliJ. I actually added it by accident but, it hides the warning which is nice I suppose? :man_shrugging: